app/buyers/views/buyers.py

```diff
@@ -307,6 +308,9 @@ def prepared_response_contents_for_brief(brief, responses):
     ess_req_names = brief.get(ESS, [])
     nth_req_names = brief.get(NTH, [])
 
+    def csv_cell_sanitize(text):
+        return re.sub(r"^(=|\+|-|@|!|\|)+", '', text.strip())
```

Comment:
They seem to encourage use of formulas, I found some more characters to remove in xlsxwriter but no helper ;)